.changeset/tiny-items-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+'@solid-design-system/components': patch
+---
+
+Fix failing test regarding sd-combobox placeholder text.
```

Comment:
```suggestion
---
---
```

As this won't change anything in the distributed files, I wouldn't bump version here.